katello/hooks/pre/10-reset_feature.rb

```diff
@@ -79,11 +79,9 @@ def empty_mongo
   if remote_host?(mongo_config[:host])
     empty_remote_mongo(mongo_config)
   else
+    Kafo::Helpers.start_service('rh-mongodb34-mongod')
```

Comment:
I have changed this to make it look unified change because this one is already having if guard and it only runs when service is running on foreman itself.